tests/cpp/test_multidevice_transformer.cpp

```diff
@@ -366,6 +441,64 @@ TEST_P(DistributedTransformerTest, MLP_Layer) {
   validate(expected_outputs, outputs);
 }
 
+TEST_P(DistributedTransformerTest, Multiheaded_Attention) {
+  auto dtype = GetParam();
+  std::unique_ptr<Fusion> fusion = std::make_unique<Fusion>();
```

Comment:
```suggestion
  auto fusion = std::make_unique<Fusion>();
```

Reply:
Prefer `auto` here because the return type of `std::make_unique<Fusion>` should be obvious to any C++11 programer even outside the project.